conmon/conmon.c

```diff
@@ -131,7 +131,7 @@ static GOptionEntry opt_entries[] =
   { "systemd-cgroup", 's', 0, G_OPTION_ARG_NONE, &opt_systemd_cgroup, "Enable systemd cgroup manager", NULL },
   { "exec", 'e', 0, G_OPTION_ARG_NONE, &opt_exec, "Exec a command in a running container", NULL },
   { "exec-process-spec", 0, 0, G_OPTION_ARG_STRING, &opt_exec_process_spec, "Path to the process spec for exec", NULL },
-  { "exit-dir", 0, 0, G_OPTION_ARG_STRING, &opt_exit_dir, "Path to the directory where exit files are written", NULL },
+  { "exit-dir", 0, 0, G_OPTION_ARG_STRING, &opt_exit_dir, "Path to the directory where exit files are written.  Not compatible with --exec", NULL },
```

Comment:
I don't see anything that prevents both exit-dir and exec from being used in the code.  Do we need an if statement to check that we didn't get both?

Reply:
> I don't see anything that prevents both exit-dir and exec from being used in the code. Do we need an if statement to check that we didn't get both?

If a caller calls for both `--exit-dir` and `--exec`, that's fine with me.  Is there a problem with that?

Reply:
I couldn't spot an issue with both flags being used at once.  If that's the case, then just remove this note here.  If there's something I'm missing and they can't be used at once, then we need an if statement.

Reply:
> If that's the case, then just remove this note here.

Ah right.  This was leftover from an earlier revision where I'd forbid both.  I'll drop this hunk.

Reply:
> If that's the case, then just remove this note here.

I've removed this change with 690cd6321 → 1390740df.